src/server_manager/model/cloud.ts

```diff
@@ -0,0 +1,97 @@
+/*
+  Copyright 2020 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+/** Enumeration of supported cloud providers. */
+export enum CloudProviderId {
+  DigitalOcean = 'DigitalOcean',
+}
+
+/** DigitalOcean REST API wrapper */
+export interface DigitalOceanApi {
```

Comment:
Let's move the API out of the model. It's a different bounded context. It's implementation details. We don't expose those concepts to the user.

It should be somewhere like infrastructure/digitalocean. You could imagine a third-party implementing that.

Reply:
As I mention in the account comments, I do think we need a model for Cloud providers though, but that's different.

Reply:
That makes sense. It's not a concept that the user is concerned about. Moved the DigitalOcean interface out of model.